testapp/tests/django/test_comma_separated_field.py

```diff
@@ -19,18 +19,18 @@ def test_csv_field_on_create():
 def test_csv_char_field():
     tag = Tag.objects.create()
 
-    assert tag.tag_categories == []
+    assert tag.tag_categories == tuple()
     tag.tag_categories = [
         "fiction",
         "history",
     ]
     tag.save()
 
     tag.refresh_from_db()
-    assert tag.tag_categories == [
+    assert tag.tag_categories == (
         "fiction",
         "history",
-    ]
+    )
```

Comment:
As seen here, you can still assign list values without breaking anything, so this change is unlikely to break existing consumers